src/PageContent/PendingAccounts/VerifyAccount.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import './VerifyAccount.css';
+import truck from '../../icons/red_truck.svg';
+import Map from '../../Map/Map';
+
+class VerifyAccount extends Component {
+    render() {
+        return (
+            <div className="verify-accounts-wrapper">
+                <h1 className="verify-header">
+                    {this.props.accountType} Account Request
+                </h1>
+                <div className="verify-map-wrapper">
+                    <Map
+                        address={this.props.agencyAddressData}
+                        height={'200px'}
+                        width={'300px'}
+                        marginRight={'30px'}
+                        marginTop={'0px'}
+                        marginBottom={'0px'}
+                        marginLeft={'10px'}
+                        display={'inline-block'}
+                    />
+                </div>
+                <div className="address-wrapper">
+                    <h2 className="subheader">{this.props.agencyName}</h2>
+                    <div>
+                        <div className="address">
+                            {this.props.agencyAddressData.street1}
+                            {this.props.agencyAddressData.street2},{' '}
+                            {this.props.agencyAddressData.city}{' '}
+                            {this.props.agencyAddressData.state}{' '}
+                            {this.props.agencyAddressData.zipcode}
+                        </div>
+                        <div className="officeNo">
+                            Office: {this.props.agencyAddressData.officeNo}
```

Comment:
Maybe only show this if there is an `officeNo` field? Looks odd if it's blank 
![image](https://user-images.githubusercontent.com/17226026/40337449-940098bc-5d24-11e8-9177-dec70b861f77.png)